packages-exp/messaging-exp/sw/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@firebase/messaging-exp",
+    "private": true,
+    "version": "0.0.900",
+    "description": "",
+    "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+    "main": "dist/index.sw.cjs.js",
+    "esm2017": "dist/index.sw.esm2017.js",
```

Comment:
You only need a `module` field pointing to the esm2017 build. The only use of this build is to create the cdn script.